app/controllers/api_controller.rb

```diff
@@ -120,7 +120,7 @@ def set_access_control_headers
   # not have these. They would instead dealing with the /api/auth
   # mechanism.
   #
-  if Vmdb::Application.config.action_controller.allow_forgery_protection
+  if Vmdb::Application.config.action_controller.allow_forgery_protection && respond_to?(:verify_authenticity_token)
```

Comment:
Nope, `skip_before_action` is the documented way to do it.

@jrafanie the previous / now-LHS conditional was a copy of the one wrapping the `protect_from_forgery` call... is it still required?